tests/test_geographic.py

```diff
@@ -58,3 +58,19 @@ def test_footprints_s2(products, fixture_path):
         expected_footprints = geojson.loads(geojson_file.read())
     # to compare unordered lists (JSON objects) they need to be sorted or changed to sets
     assert set(footprints) == set(expected_footprints)
+
+
+@pytest.mark.vcr
+@pytest.mark.fast
```

Comment:
`@pytest.mark.scihub` marker would be a better fit here and in the next test. `fast` is intended for tests that don't try to access the internet.